components/sysutils/ngrep/ngrep.p5m

```diff
@@ -13,6 +13,8 @@
 # Copyright 2016 Adam Stevko
 #
 
+<transform file path=usr.*/man/.+ -> default mangler.man.stability uncommitted>
```

Comment:
For some reason, we have it in many manifests... and carry over into new ones too :)